src/schema.js

```diff
@@ -29,27 +48,12 @@ const RootQuery = new GraphQLObjectType({
   fields: {
     item: {
       type: ItemType,
-      args: { id: { type: GraphQLInt } },
-      resolve: async (root, args) => {
-        const url = `https://hacker-news.firebaseio.com/v0/item/${args.id}.json`
-        const { data } = await get(url)
-        return data
-      }
+      args: { id: { type: new GraphQLNonNull(GraphQLInt) } },
```

Comment:
https://medium.com/@calebmer/when-to-use-graphql-non-null-fields-4059337f6fc8

Reply:
cool! that made for some really good insights